packages/protocol/scripts/bash/release-on-devchain.sh

```diff
@@ -63,7 +63,8 @@ yarn ts-node scripts/check-backward.ts sem_check --old_contracts $BUILD_DIR/cont
 
 # From make-release.sh
 echo "- Deploy release of current branch"
-yarn truffle exec --network development ./scripts/truffle/make-release.js --build_directory build/ --report report.json --proposal proposal.json --initialize_data example-initialize-data.json
+INITIALIZATION_FILE=`ls -t releaseData/initializationData/* | head -n 1`
```

Comment:
would love for this to be parameterized in 1 source location in future
ie CI config reads from here or a `protocol/env`